config/analytics_pii.yml

```diff
@@ -45,3 +45,9 @@ shared:
   - email
   :international_relocation_payments_eligibilities:
   - passport_number
+  - school_headteacher_name
+  - school_name
+  - school_address_line_1
+  - school_address_line_2
+  - school_city
+  - school_postcode
```

Comment:
Please move these to `analytics_blocklist.yml` - see https://dfedigital.atlassian.net/browse/CAPT-1741

Reply:
👍